books/free-programming-books-langs.md

```diff
@@ -2442,6 +2442,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Developing Time-Oriented Database Applications in SQL](https://www2.cs.arizona.edu/~rts/tdbbook.pdf) - Richard T. Snodgrass (PDF)
 * [Essential SQL](https://www.programming-books.io/essential/sql) - Krzysztof Kowalczyk and  Stack Overflow Documentation project (HTML)
 * [Introduction to SQL](https://github.com/bobbyiliev/introduction-to-sql) - Bobby Iliev (Markdown, PDF)
+* [Oracle Database Book](https://goalkicker.com/OracleDatabaseBook/) Compiled from StackOverflow Documentation (PDF)
```

Comment:
does this link work for you? https://goalkicker.com/OracleDatabaseBook/OracleDatabaseNotesForProfessionals.pdf ?
Don't invent titles! Use the title given by the resource.

Reply:
you don't want to use the link direct to pdf? or did it redirect you?

Reply:
the website is a hot mess of ads; the pdf is fine. just one link please.

Reply:
ok then i will just keep the pdf file link.